toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -32,7 +32,7 @@ install_one_toolchain_rpm () {
     fi
 
     echo "Found full path for package $1 in $rpm_path: ($full_rpm_path)" >> "$chroot_log"
-    rpm -i -v --nodeps --noorder --force --root "$chroot_builder_folder" --define '_dbpath /var/lib/rpm' "$full_rpm_path" &>> "$chroot_log"
+    rpm -i -v --nodeps --noorder --force --noscripts --root "$chroot_builder_folder" --define '_dbpath /var/lib/rpm' "$full_rpm_path" &>> "$chroot_log"
```

Comment:
Can we special-case the `noscripts` flag to only apply to the msopenjdk RPM? I believe applying this to all RPMs will prevent things like our ldconfig scriptlets from running.

Reply:
good idea, I've updated the script to special-case this package